Description: Let debian/ruby-tests.rb handle the test suite loading
 Was causing FTBFS because ext was loaded.
Origin: vendor
Author: Lucas Nussbaum <lucas@debian.org>
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=634438
--- a/tests/test_helper.rb
+++ b/tests/test_helper.rb
@@ -2,13 +2,10 @@ gem 'json', File.read('VERSION').chomp
 
 case ENV['JSON']
 when 'pure'
-  $:.unshift 'lib'
   require 'json/pure'
 when 'ext'
-  $:.unshift 'ext', 'lib'
   require 'json/ext'
 else
-  $:.unshift 'ext', 'lib'
   require 'json'
 end
 
